Cargo.toml

```diff
@@ -79,6 +86,7 @@ cfg_aliases = "0.2.1"
 bitflags = "2"
 cursor-icon = "1.1.0"
 dpi = { version = "0.1.1", path = "dpi" }
+keyboard-types = { git = "https://github.com/pyfisch/keyboard-types.git", branch = "main" }
```

Comment:
This PR is now ready to be reviewed, though note that v0.8 isn't released yet, so we shouldn't merge it before it is.

Reply:
Yeah, we certainly should, so I've converted to draft for now.